spring-integration-core/src/test/java/org/springframework/integration/configuration/EnableIntegrationTests.java

```diff
@@ -296,6 +313,9 @@ public void testAnnotatedServiceActivator() {
 		assertEquals(10L, TestUtils.getPropertyValue(trigger, "period"));
 		assertFalse(TestUtils.getPropertyValue(trigger, "fixedRate", Boolean.class));
 
+		// Markers to investigate the failures on CI
+		System.out.println("----SEND Message to 'input' channel----");
```

Comment:
Understood; but I'd prefer to change them, just in case we forget about them.

I confirmed that the tangle is fixed; merging (with logger).